bundles/org.openhab.binding.mqtt.homeassistant/src/test/java/org/openhab/binding/mqtt/homeassistant/internal/component/JSONSchemaLightTests.java

```diff
@@ -76,7 +76,7 @@ public void testRgb() throws InterruptedException {
         publishMessage("zigbee2mqtt/light/state", "{ \"color\": {\"r\": 10, \"g\": 20, \"b\": 30 } }");
         assertState(component, Light.COLOR_CHANNEL_ID, HSBType.fromRGB(10, 20, 30));
         publishMessage("zigbee2mqtt/light/state", "{ \"brightness\": 255 }");
-        assertState(component, Light.COLOR_CHANNEL_ID, new HSBType("210,67,100"));
+        assertState(component, Light.COLOR_CHANNEL_ID, new HSBType("209.99999999999999,66.6666666666666700,100"));
```

Comment:
I'm not convinced that this is a good solution. We would have to adapt the test again when we decide to limit the precision to scale 10 or so. It would be better to adjust `assertState` to something like


```
   State actualState = component.getChannel(channelId).getState().getCache().getChannelState();
   if (actualState instanceof HSBType hsb) {
       assertThat(hsb.closeTo((HSBType) state, 0.01), is(true));
   } else { 
       assertThat(actual, is(state));
   }
```